doc/theory/glossary.tex

```diff
@@ -307,76 +304,69 @@
-	
-	The $p$\textit{-norm} of a 
-	% \hyperref[persistence_landscape]{persistence landscape}
-	persistence landscape $\lambda = \{\lambda_k\}_{k \in \mathbb N}$ is defined to be
-	
-	\begin{equation*}
-	||\lambda||_p = \left( \sum_{i \in \mathbb N} ||\lambda_i||^p_p \right)^{1/p}
-	\end{equation*}
-	whenever the right hand side exists and is finite.
+	%\hyperref[persistence diagram] {persistence diagram}
+	persistence diagram. Its \textit{persistence landscape} is the set $\{\lambda_k\}_{k \in \mathbb N}$ of functions	
+	\begin{equation*}	
+	\lambda_k : \mathbb R \to \overline{\mathbb R}	
+	\end{equation*}	
+	defined by letting $\lambda_k(t)$ be the $k$-th largest value of the set $\{\Lambda_i(t)\}_ {i \in I}$ where	
+	\begin{equation*}	
+	\Lambda_i(t) = \left[ \min \{t-b_i, d_i-t\}\right]_+	
+	\end{equation*}	
+	and $c_+ := \max(c,0)$. The function $\lambda_k$ is referred to as the \textit{$k$-layer of the persistence landscape}.
+	
+	We describe the graph of each $\lambda_k$ intuitively. For each $i \in I$, draw an isosceles triangle with base the interval $(b_i, d_i)$ on the horizontal $t$-axis, and sides with slope 1 and $-1$. This subdivides the plane into a number of polygonal regions. Label each of these regions by the number of triangles containing it. If $P_k$ is the union of the polygonal regions with values at least $k$, then the graph of $\lambda_k$ is the upper contour of $P_k$, with $\lambda_k(a) = 0$ if the vertical line $t=a$ does not intersect $P_k$.
+	
+	The persistence landscape construction defines a
+	%\hyperref[vectorization, kernel and amplitude]{vectorization}
+	vectorization of the set of persistence diagrams with target the vector space of real-valued function on $\mathbb N \times \mathbb R$. For any $p = 1,\dots,\infty$ we can restrict attention to persistence diagrams $D$ whose associated persistence landscape $\lambda$ is
+	%\hyperref[l^p norm]{$p$-integrable}
+	$p$-integrable, that is to say,	
+	\begin{equation} \label{equation:persistence_landscape_norm}	
+	||\lambda||_p = \left( \sum_{i \in \mathbb N} ||\lambda_i||^p_p \right)^{1/p}	
+	\end{equation}	
+	where
+	\begin{equation*}	
+	||\lambda_i||_p = \left( \int_{\mathbb R} \lambda_i^p(x)\, dx \right)^{1/p}	
+	\end{equation*}	
+	is finite. In this case we refer to \eqref{equation:persistence_landscape_norm} as the
+	%\hyperref[vectorization, kernel and amplitude]{amplitude}
+	\textit{landscape} $p$-\textit{amplitude} of $D$.
 
-	\paragraph{\\ References:} \cite{stein2011functional, bubenik2015statistical}
+	\paragraph{\\ References:} \cite{bubenik2015statistical}
 	
 	\subsection*{Weighted silhouette} \label{weighted_silhouette}
 	
-	Let $D = {(b_i, d_i)}_{i \in I}$ be a
-	% \hyperref[persistence_diagram] {persistence diagram}
-	persistence diagram. A \textit{weighted silhouette} associated to $D$ is a continuous function $\phi : \mathbb R \to \mathbb R$ of the form
+	Let $D = \{(b_i, d_i)\}_{i \in I}$ be a
+	%\hyperref[persistence diagram] {persistence diagram}
```

Comment:
Ditto as above. Also there is here a whitespace between brackets and parentheses.